blueprints/data-solutions/data-platform-foundations/demo/datapipeline_dc_tags.py

```diff
@@ -283,7 +283,7 @@
           'datasetId': DWH_CURATED_BQ_DATASET,
           'tableId': 'customer_purchase'
         },
-        'writeDisposition':'WRITE_APPEND',
+        'writeDisposition':'WRITE_TRUNCATE',
```

Comment:
I think this is unintended change as it rollbacks #1533 (+ other changes in *py files here)

Reply:
good catch

Reply:
fixed